add_events.py

```diff
@@ -112,8 +112,12 @@ def main():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            if (data[day][time][0] in subjects.keys()):
-                event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+            # if (data[day][time][0] in subjects.keys()):
+            if (data[day][time][2] < 3):
```

Comment:
If the L and T are not zero (such as English which has 3-0-2, if I am not wrong, then there are two slots that are given to one course. I am not sure how GYFT currently handles this.

@athityakumar More info on this one please.

cc @nishnik @DefCon-007